CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,141 @@
+# Release notes for 1.9.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.8.0
+
+## Changes by Kind
+
+### Feature
+- Update recover from expansion failure to use newer APIs (#270, @gnufied)
```

Comment:
Also note that this will require 1.28.0.  And make the recommended version 1.28 in the README